conslogging/conslogging.go

```diff
@@ -467,11 +468,27 @@ func (cl ConsoleLogger) color(c *color.Color) *color.Color {
 	return noColor
 }
 
+var (
+	// githubRegExp Matches :2dd88e53f2e59e96ec1f9215f24a3981e5565edf+ in a target.
+	// 	Prefix containing hash may resemble: g/e/hello-world:2dd88e53f2e59e96ec1f9215f24a3981e5565edf+base
+	githubRegExp = regexp.MustCompile(`:[a-zA-Z0-9]*\\+`)
+)
+
 func prettyPrefix(prefixPadding int, prefix string) string {
 	if prefixPadding == NoPadding {
 		return prefix
 	}
 
+	// If the prefix contains a GitHub hash - shorten it
+	gitHashPart := githubRegExp.FindString(prefix)
+	resultingHashLength := 9 // 1 for ':', 1 for '+', 7 for hash
+	if len(gitHashPart) > resultingHashLength {
+		hash := gitHashPart[1:]
+		hash = hash[:len(gitHashPart)-1]
+		hash = hash[:7]
```

Comment:
Are we truncating the string twice here (L487)? Can that be done in one op?

Reply:
done